DI-Policies.md

```diff
@@ -0,0 +1,51 @@
+# Statement
+
+Historically, the wedding industry has been massively lacking in areas of representation and equity, especially across race, gender, and sexual orientation. The fact that interracial marriage wasn’t fully legal in the U.S. until 1967, and same sex marriage wasn’t legal until 2015, has been reflected in wedding industry practices and imagery since its beginning. And to this day, it’s painfully clear just how racist, sexist, homophobic, and generally prejudiced the industry still is.
+
+It’s up to wedding vendors and industry leaders to correct this. We have a responsibility to create spaces and content where all people feel seen, safe, and valued. 
+
+It’s important to me as a wedding business owner to not only implement policies that address these issues, but also hold myself accountable to those policies. So, I’m sharing this document publicly as one part of my own efforts to do better and be better. This not an “arrival”, but rather part of an ongoing process that will evolve and change over the lifetime of my business as I learn and iterate to create more inclusive, welcoming, and safe spaces for all folx.
+
+# Specific policies
+
+These are the specific actions I commit to taking as integral parts of how I do business moving forward. 
+
++ Listening + amplifying
+
+   + Listen to and amplify the stories and experiences of marginalized groups in the industry, and out of it.
+
+   + Speak up when witness to problematic statements, situations, and practices.
+
+   + Own my own mistakes when - not if - I make them, make reparations, and do better next time.
+
++ Regular audits of website and social media feeds
+
+   + Quarterly for website.
+
+   + Monthly for social.
+
++ Vetting subcontractors + partners
+
+   + Interview second shooters, editors, and any others I work with or hire to ensure inclusive practices and the comfort/safety of my clients.
+
++ Continuing education 
+
+   + Minimum two PAID workshops a year (pay Black folx and other marginalized individuals for their work).
+
+   + Before participation in any styled shoots, workshops, or other events, inquire into their own D+I policies. 
+
++ Networking + referrals
+
+   + Regularly and consistently seek out, work with, and refer underrepresented vendors.
+
++ Advertising + outreach
+
+   + Do business with and support sites, publications, and organizations that are focused on social justice and equity in the wedding industry.
```

Comment:
> Do business with and support

I'm struggling to see how this is different from points you've already mentioned. Perhaps this could be reworded as 'Promote and/or partner with'

Reply:
@bryanaknight I definitely see what you're saying. This is super unclear. I was primarily trying to make a point about wanting to do better about vetting where *I'm* advertising specifically. I want to ensure I'm not paying to be in a directory/on a website that has proven historical habits of racist, homophobic, or other issues and, rather, intentionally advertise on sites/with companies that have done good work in helping amplify and support underrepresented groups in the industry.

I think I'm understanding that this section seems to read as me helping promote/advertise for others, whereas I really mean to say that I'll be mindful of where I'm advertising my own services. So, maybe I really need to change the header here to make that part more clear. Maybe something like `Advertising spend + outreach` and then changing the bullet to be more clear too...something like `Spend advertising dollars with sites, publications, and organizations that are focused on...`